### DISCUSSION TITLE
ElevenLabs connector

### DISCUSSION THREAD

**Discussion by MithrilMan** [microsoft/semantic-kernel#6212] [Ideas]
I haven't found an ElevenLabs connector to use their TTS, is this a needed feature where I can spend time implementing a connector for and push a PR?
Or the approach would be to create a repo with its own nuget package out of this repository?

**Comment by sophialagerkranspandey** [comment]
Thanks for reaching out. We'd recommend you create your own repo as we don't have plans to create a connector for ElevenLabs at this time.
